reference/4.0/PSDiagnostics/Enable-PSWSManCombinedTrace.md

```diff
@@ -1,38 +1,50 @@
 ---
-ms.date:  06/09/2017
-schema:  2.0.0
-locale:  en-us
-keywords:  powershell,cmdlet
-external help file:  PSDiagnostics-help.xml
+external help file: PSDiagnostics-help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: PSDiagnostics
+ms.date: 06/09/2017
```

Comment:
Still wrong here, too.

Reply:
I thought I got all of these. Found and fixed.